### DISCUSSION TITLE
templater files

### DISCUSSION THREAD

**Discussion by jmscottimd** [BryanJenksCommunity/FAQ#117] [Q&A] [answered]
i am not sure if i have an issue on my end or if the issue is that I have the wrong version of the vault files. i sent a request for the updated files. should there be a `md` file for the default 🌱  template?  i do not see one in the directory and when I create a note without the character the 🌱 is not applied.

**Comment by tallguyjenks** [comment, marked as answer]
are you inserting the meta template with templater and not the core template plugin?

**Reply by jmscottimd** [reply]
updating the files solved the problem. thank you for sharing the files and responding.

**Reply by tallguyjenks** [reply]
Yay! Glad it was fixed :)
